SUIApp/SUIApp/TabBarView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ContentView.swift
+//  SUIApp
+//
+//  Created by Vladka on 16.01.2023.
+//
+
+import SwiftUI
+
+/// Таббар
+struct TabBarView: View {
+    
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let vpnTitleText = "VPN"
+        static let rateTitleText = "Rate"
+        static let basketTitleText = "Basket"
+        static let vpnSystemImageNameText = "lock"
+        static let rateSystemImageNameText = "dollarsign"
+        static let basketSystemImageNameText = "basket"
+        static let zeroNumber = 0
+        static let oneNumber = 1
+        static let twoNumber = 2
+    }
+ 
+    var body: some View {
+        tabBarView
+    }
+    
+    // MARK: - Private Property
+    
+    @State private var selectedIndex = Constants.zeroNumber
+    @StateObject private var purchaseModel = PurchaseModel()
+    private var tabBarView: some View {
+        TabView(selection: $selectedIndex) {
+            VPNView()
+                .tabItem {
+                    Label(Constants.vpnTitleText, systemImage: Constants.vpnSystemImageNameText)
+                }
+                .tag(Constants.zeroNumber)
+            PurchaseView(tabSelection: $selectedIndex)
+                .tabItem {
+                    Label(Constants.rateTitleText, systemImage: Constants.rateSystemImageNameText)
+                }
+                .tag(Constants.oneNumber)
+            BasketView(tabSelection: $selectedIndex)
+                .tabItem {
+                    Label(Constants.basketTitleText, systemImage: Constants.basketSystemImageNameText)
+                }
+                .tag(Constants.twoNumber)
+        }
+        .environmentObject(purchaseModel)
+    }
+}
+
+struct TabBarView_Previews: PreviewProvider {
```

Comment:
```suggestion
// MARK: - PreviewProvider

struct TabBarView_Previews: PreviewProvider {
```

Сюда тоже можно марку добавить судя по скрину из важного